packaging/installer/methods/packages.md

```diff
@@ -13,15 +13,12 @@ and fail if it cannot do so.
 > **Note**
 >
 > In July 2022, we switched hosting of our native packages from Package Cloud to self-hosted repositories.
-> We still maintain the Package cloud repositories, but they are not guaranteed to work and may be removed
-> without prior warning.
->
-> When selecting a repository configuration package, note that the version 2 packages provide configuration for
-> our self-hosted repositories, and then version 1 packages provide configuration for Package Cloud.
+> Until late 2024 we continued to provide packages via Package Cloud, but we have since then switched to only
+> providing packages via our own repositories.
```

Comment:
```suggestion
> Until late 2024 we continued to provide packages via Package Cloud, but we have since then switched to only
> providing packages via our repositories.
```